test/statmodels.jl

```diff
@@ -34,5 +34,13 @@ x3   0.453058  0.72525 0.999172 0.5567
 try
     throw(ConvergenceException(10))
 catch ex
-    @test sprint(showerror, ex) == "failure to converge after 10 iterations"
+    @test sprint(showerror, ex) == "failure to converge after 10 iterations."
 end
+
+try
```

Comment:
Sorry, I misread. Though you could use `@test_throws` just like you do below for a slightly more compact code.